schema.sql

```diff
@@ -1,5 +1,4 @@
+
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+ CREATE TABLE animals (id integer ,name varchar , date_of_birth date , escape_attempts integer , neutered boolean , weight_kg decimal );
```

Comment:
- [ ] Kindly, add a specified length for the varchar column `name`.
for example : `name varchar(200),`
- [ ] [OPTIONAL] Also, I'll suggest you to make the column id NOT NULL and  as the primary key.